AVInput/AVInput.json

```diff
@@ -99,6 +146,454 @@
                     "success"
                 ]
             }
+        },
+        "getInputDevices": {
+            "summary": "Returns an array of available HDMI/Composite Input ports.\n \n### Events\n \nNo Events.",
+            "result": {
+                "type": "object",
+                "properties": {
+                    "devices": {
+                        "$ref": "#/definitions/devices"
+                    },
+                    "success": {
+                        "$ref": "#/common/success"
+                    },
+                    "typeOfInput": {
+                        "$ref": "#/definitions/typeOfInput"
+                    }
+                },
+                "required": [
+                    "devices",
+                    "success"
+                ]
+            },
+            "errors": [
+                {
+                    "description": "User name or room name was invalid",
+                    "$ref": "#/common/errors/badrequest"
+                }
+            ]
+        },
+        "getEdidVersion":{
+            "summary": "(Version 2) Returns the EDID version.\n \n### Events\n \nNo Events.",
+            "params": {
+                "type":"object",
+                "properties": {
+                    "portId":{
+                        "$ref": "#/definitions/portId"
+                    }
+                },
+                "required": [
+                    "portid"
+                ]
+            },
+            "result": {
+                "type": "object",
+                "properties": {
+                    "edidVersion": {
+                        "summary": "The EDID version",
+                        "type": "string",
+                        "example": "HDMI2.0"
+                    },
+                    "success": {
+                        "$ref": "#/common/success"
```

Comment:
please do not use XRE Service Manager legacy param "success"

Reply:
Removed success param